prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.html

```diff
@@ -25,7 +25,7 @@
           <ng-container *ngFor="let careSetting of careSettings.controls; let i = index;"
                         [formGroupName]="i">
 
-            <app-form-icon-group [show]="careSettings.length > 1"
+            <app-form-icon-group [show]="careSettings?.length > 1"
```

Comment:
This looks a bit odd.  I would suspect that it isn't possible to have `careSettings` be null or undefined since it would have a default of empty array in the form state service.  I think this is likely something in the test that isn't ready yet, and not a failing of the component.